src/EnergyPlus/DataGlobalConstants.hh

```diff
@@ -102,24 +101,7 @@ namespace DataGlobalConstants {
         ElectricityNet,
         SolarWater,
         SolarAir,
-        SO2,
-        NOx,
-        N2O,
-        PM,
-        PM2_5,
-        PM10,
-        CO,
-        CO2,
-        CH4,
-        NH3,
-        NMVOC,
-        Hg,
-        Pb,
-        NuclearHigh,
-        NuclearLow,
-        WaterEnvironmentalFactors,
         CarbonEquivalent,
-        Source,
```

Comment:
Can someone explain to me what physical thing `Source` is?

Reply:
Source energy, i.e. the amount of energy used to produce a unit of energy at the meter. e.g. How much gas energy went into the power plant to produce a kWh of electricity at the meter (including losses, etc.). This should probably be renamed to `SourceEnergy`.

Reply:
Aha. That makes sense. Strange that SourceEnergy is so aggregated whereas everything else is so disaggregated.

Reply:
It's the original measure from long ago for making apples-to-apples comparisons of electricity vs gas (or other end-use fuel types). $ are another proxy, and now CarbonEquivalent is a biggie, which is aggregated back to the source as well.